core/domain/user_query_jobs_one_off.py

```diff
@@ -139,7 +174,8 @@ def map(user_settings_model):
 
         predicates = constants.EMAIL_DASHBOARD_PREDICATE_DEFINITION
         for predicate in predicates:
-            if getattr(query_model, predicate['backend_attr']) is not None:
+            value = getattr(query_model, predicate['backend_attr'])
+            if value is not None and value is not False:
```

Comment:
Why is the ``is not False`` check needed now? Given that the value can be different types, this seems slightly hacky.

Reply:
I have changed it to predicate['default_value']. Done.